### PR TITLE
Added example notebook for tracing and deploying CLIP model to opensearch

### PR DESCRIPTION
**Pull request by patrickbarnhart**
### Description
Uploaded notebook provides a walkthrough for users to trace, register, and deploy a CLIPTextModel from a local file. CLIPTextModel can be used with the Neural Search plugin to generate embeddings of documents and ingest time and of user queries at search time. 
 
### Issues Resolved
N/A - this is documentation
 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

Signed-off-by: Patrick Barnhart <patrickAbarnhart@gmail.com>

### CODE REVIEW

File: docs/source/examples/demo_deploy_cliptextmodel.ipynb [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      "\u001b[1;31mNameError\u001b[0m: name 'ml_client' is not defined"
+     ]
+    },
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
```
Comment (dhrubo-os) [inline docs/source/examples/demo_deploy_cliptextmodel.ipynb:225]: Are we keeping this?

File: docs/source/examples/demo_deploy_cliptextmodel.ipynb [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    "* Record the model ID from the output of the next cell\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
```
Comment (dhrubo-os) [inline docs/source/examples/demo_deploy_cliptextmodel.ipynb:212]: Looks like there's an error?

Comment (patrickbarnhart) [reply docs/source/examples/demo_deploy_cliptextmodel.ipynb:212]: Accidentally ran this cell before loading the packages which caused this error. It should not be part of the commit - the notebook runs if executed in sequential order

File: docs/source/examples/demo_deploy_cliptextmodel.ipynb (hunk 1 above)
Comment (patrickbarnhart) [reply docs/source/examples/demo_deploy_cliptextmodel.ipynb:225]: No, this error text should be removed from the commit

**Comment by patrickbarnhart** [comment]
Notebook is updated to resolve errors. Latest commit has updated notebook

**Review by dhrubo-os — APPROVED** [review]

**Comment by patrickbarnhart** [comment]
Added note to CHANGELOG

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**